superset/models/core.py

```diff
@@ -419,6 +450,13 @@ def _get_sqla_engine(
                 sqlalchemy_url, params, effective_username, security_manager, source
             )
 
+        if ssh_tunnel_server:
+            # update sqlalchemy_url
+            url = make_url_safe(sqlalchemy_url)
+            sqlalchemy_url = url.set(
+                host="127.0.0.1", port=ssh_tunnel_server.local_bind_port
```

Comment:
can we define this host value as a constant somewhere?

Reply:
Should be fixed as part of: https://github.com/apache/superset/pull/22306